sdk/swagger.json

```diff
@@ -1187,6 +1306,48 @@
         "workspaceCollectionName": {
           "type": "string",
           "description": "The workspace collection name"
+        },
+        "displayName": {
```

Comment:
Didn't Mahir already checked it in to master? Why does it show-up within the diff?